### ISSUE TITLE
How to use CodeGraphAnalyzer for dumping CSV file

### ISSUE THREAD

**Issue by nguyendacthienngan** [for-just-we/CodeGraphAnalyzer#2]
As I can't install Joern old version, I want to try your CodeGraphAnalyzer to analyze. But I cannot find any guidelines for using it? Can you provide some steps? 
Also, I'm wondering [Joern.zip](https://github.com/for-just-we/VulDetectArtifact/blob/main/preprocess/joern.zip) file is for what?

**Comment by for-just-we** [comment]
> As I can't install Joern old version, I want to try your CodeGraphAnalyzer to analyze. But I cannot find any guidelines for using it? Can you provide some steps? Also, I'm wondering Joern.zip file is for what?

dumping CSV? like print basic information of edges? You can refer [PrintEdgeVisitor](https://github.com/for-just-we/CodeGraphAnalyzer/blob/master/src/main/java/org/example/visitors/PrintEdgeVisitor.java)

**Comment by nguyendacthienngan** [comment]
As you said "Our main pipeline for extract code graphs is done by [Joern](https://github.com/joernio/joern). specifically, we use old version of [Joern](https://github.com/octopus-platform/joern) because we find the new version is too hard to use, the old version will dump csv files, so our preprocess is mainly parsing the csv files." on [PREPROCESS ](https://github.com/for-just-we/VulDetectArtifact/tree/main/preprocess).
You used joern-parse to get the CSV file. Respectively, can I use CodeGraphAnalyzer to do the same things?

**Comment by for-just-we** [comment]
> As you said "Our main pipeline for extract code graphs is done by [Joern](https://github.com/joernio/joern). specifically, we use old version of [Joern](https://github.com/octopus-platform/joern) because we find the new version is too hard to use, the old version will dump csv files, so our preprocess is mainly parsing the csv files." on [PREPROCESS ](https://github.com/for-just-we/VulDetectArtifact/tree/main/preprocess). You used joern-parse to get the CSV file. Respectively, can I use CodeGraphAnalyzer to do the same things?

Yes, but you may need to modify the code, you just need to print the content of AST node in the format you like. You can refer [PrintEdgeVisitor](https://github.com/for-just-we/CodeGraphAnalyzer/blob/master/src/main/java/org/example/visitors/PrintEdgeVisitor.java) to see how to print the content.
